enhancements/proposed/20200501-json-output/start-json-output.md

```diff
@@ -0,0 +1,231 @@
+# Add JSON Output to Minikube Start
```

Comment:
The same plan applies to `minikube stop` and maybe some other commands? Anything that could stall should be able to report what step it stalled on, I guess.

Reply:
This should be extendable for other commands -- each command will have different associated "Steps" for the logs, so we're starting with figuring it out for `start`